witness/README.md

```diff
@@ -1,3 +1,116 @@
+> stack build disciplina-witness -- --help
+```
+
+```
+Usage: dscp-witness [--version] [--debug] [--log-config FILEPATH]
+                    [--log-dir FILEPATH] [--db-path FILEPATH]
+                    (--peer HOST:PORT1:PORT2)* --bind HOST:PORT1:PORT2
+                    [--bind-internal HOST:PORT1:PORT2]
+                    [--witness-keyfile FILEPATH] [--witness-gen-key]
+                    [--witness-keyfile-pass PASSWORD] [--comm-n INTEGER]
+                    [--comm-sec BYTESTRING] --witness-listen HOST:PORT
+                    (-c|--config FILEPATH) --config-key STRING
+
+Available options:
+  -h,--help                Show this help text
+  --version                Show version.
+  --debug                  Switch default logging level from Info to Debug
+  --log-config FILEPATH    Path to logger configuration.
+  --log-dir FILEPATH       Path to logs directory.
+  --db-path FILEPATH       Path to database directory for witness node.
+  --peer HOST:PORT1:PORT2  Peer(s) we should connect to.
+  --bind HOST:PORT1:PORT2  Host/ports to bind on, also the public address we
+                           share with other nodes. Two ports are needed for
+                           hosting pub/sub ZMQ sockets accordingly.
+  --bind-internal HOST:PORT1:PORT2
+                           Overrides address to bind on, still the --bind value
+                           must be addressable and designates publically visible
+                           address. Use this option when the node is beyond NAT.
+  --witness-keyfile FILEPATH
+                           Path to the secret key of the witness.
+  --witness-gen-key        Generate the key and write it to
+                           'witness-keyfile-path' path. If file already exists
+                           at given path, secret in it used.
+  --witness-keyfile-pass PASSWORD
+                           Password of secret key.
+  --comm-n INTEGER         Committee participant index. In event of secret key
+                           file generation, will be used to derive the secret.
+  --comm-sec BYTESTRING    Committee secret key. Common key for the core nodes
+                           which serves as root key in generation of
+                           participants' secrets.
+  --witness-listen HOST:PORT
+                           Host/port for serving Witness API.
+  -c,--config FILEPATH     Path to configuration file.
+  --config-key STRING      Configuration key. Name of top-level section of
+                           configuration to use.
+```
+
+### Configuration
+
+Configuration is a `.yaml` file containing parameters common for all nodes.
+See example in [configuration.yaml](../configuration.yaml).
+
+Top-level entries describe a single setting, be it local demo or production cluster.
+Yaml-key of such entry is refered as "configuration key".
+
+Each setting contains following parameters:
+
+* `homeParam` - application home directory. It is created on launch automatically
```

Comment:
These are core parameters. Configuration description should be separate, I think, and it must be located in the repo root.

Reply:
Right on the repo main page, are you sure? :thinking:

Reply:
Well, technically I'd like it to be a `man` page or something, but yes, it should be in the repo root or in `docs/` or whatever. Not in `witness`.